mdpow/equil.py

```diff
@@ -33,8 +33,9 @@
 
 import os, errno
 import shutil
-import cPickle
+
 import MDAnalysis as mda
+from six.moves import cPickle as pickle
```

Comment:
minor thing: leave the import in the same position as before, we roughly order them by
1. standard library
2. common libraries/compatibility
3. "big" imports like numpy, MDAnalysis
4. own imports